mentors/scala/b.json

```
@@ -1 +1,10 @@
-[]
+[
+  {
+  "github_username": "leobenkel",
```

```suggestion
    "github_username": "leobenkel",
```

indent other lines like this too.